myprotein.py

```diff
@@ -4,8 +4,10 @@
 import itertools
 import json
 import operator
+# noreorder pylint: disable=wrong-import-order
```

Comment:
```suggestion
# noreorder pylint doesn't work correctly locally and in TravisCI env. This can be removed when isort releases updated version
# noreorder pylint: disable=wrong-import-order
```